seismic-app/Dockerfile

```diff
@@ -9,8 +9,13 @@ ENV PATH /app/node_modules/.bin:$PATH
 
 # install and cache app dependencies
 COPY package*.json ./
-RUN npm install
-RUN npm install react-scripts@3.2.0 -g
+
+COPY . ./
+
+COPY docker-entrypoint.sh /usr/local/bin/docker-entrypoint
+RUN chmod +x /usr/local/bin/docker-entrypoint
+
+ENTRYPOINT ["docker-entrypoint"]
```

Comment:
These are the changes from the gist
Docker runs everything and I get a very nice `Starting seismic-app ... done` at the end, but I still can't connect to the app

Reply:
OK, I pull your branch of the code and I'll do some troubleshooting

Reply:
Thanks a lot! 🥇

Reply:
@tudoramariei I got it to run. You just have to wait for the `npm ci` and `react-scripts start` commands to finish. For me, after docker-compose told the the container is up and running (`docker-compose ps`) it took another 2 minutes (Mac OS, older Mac). 
<img width="760" alt="image" src="https://user-images.githubusercontent.com/153277/69433929-3fa67700-0d45-11ea-88b8-6a02a8671394.png">

Reply:
One more issue I see here (also on a Mac, 2016 one)
The container seems to die in 5 seconds after it starts on my computer...

Reply:
I think I found the problem. When looking at the docker logs for that container I saw that it crashes at `npm ci`.  I ran `npm install ci` on my machine and after 2 minutes the container is still alive 🙌 
I think adding `RUN npm install ci` in `Dockerfile` or `npm install ci` in `docker-entrypoint.sh` would be the solution. Which one would you recommend?

Reply:
`npm ci` it's a native npm command since 5.7,  and it should work without installing anything else. You might have some issues with that since you already have a `node_modules` directory generated by running npm command directly on the host system (Mac in your case).
I used `npm ci` instead of the regular `npm install` because I don't `npm install` to change the lock file (`packages-lock.json`). Usually I use `yarn` which has a flag special for this.
Here we could just replace `npm ci` with `npm install` and be done with it.

Reply:
`npm install` seems to do the job just right
It's time to clean up the PR and get it ready for a `merge`